coursemapper-kg/recommendation/app/services/course_materials/recommendation/resource_recommender_helper.py

```
@@ -0,0 +1,574 @@
+import json
+from datetime import datetime, timedelta
```

Ruff (0.12.2)

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)